src/libraries/Native/Unix/CMakeLists.txt

```diff
@@ -5,6 +5,10 @@ project(CoreFX C)
 
 include(${CLR_ENG_NATIVE_DIR}/configuretools.cmake)
 
+if(CLR_CMAKE_TARGET_IOS)
+    cmake_minimum_required(VERSION 3.14.5)
```

Comment:
Shall we comment why this is required for iOS?
@jkotas, if this is the minimum version required, which only differs by minor version, should we rev the minimum version to this? `3.14.2` by itself is not popular in distos and brew on macOS is already revved to `3.15.3`. We have tool validation in shell scripts, so it kind of does not make much sense to have one-off exception.

Reply:
I agree that it would be best to delete this from here.

Reply:
3.14.5 contains a few bug fixes for iOS, I can add a comment. Bumping the min CMake used by everyone else seemed like a bigger change so I didn't want to do it in my initial iOS PR.

If we can bump it everywhere that seems better.

Reply:
The documented repo requirements are 3.15.5 or newer:

https://github.com/dotnet/runtime/blob/master/docs/workflow/requirements/macos-requirements.md#toolchain-setup

Reply:
btw, the minimum version is mostly only specified in top-level script files these days:

```sh
$ git grep "_minimum_required\|3\.14\.2"  # search for _minimum_required OR 3.14.2

docs/workflow/requirements/linux-requirements.md:Building the repo requires CMake 3.14.2 or newer on Linux. Add Kitware's APT feed to your configuration for a newer version of CMake. See their instructions at <https://apt.kitware.com/>. Also, add LLVM/s APT feed to your configuration for a newer version of CMake. See their instructions as <http://apt.llvm.org/>.
eng/native/build-commons.sh:    if [[ "$(version "$cmake_version")" -lt "$(version 3.14.2)" ]]; then
eng/native/build-commons.sh:        echo "Please install CMake 3.14.2 or newer from http://www.cmake.org/download/ or https://apt.kitware.com and ensure it is on your path."; exit 1;
global.json:    "cmake": "3.14.2",
src/coreclr/CMakeLists.txt:cmake_minimum_required(VERSION 3.14.2)
src/coreclr/src/pal/tests/CMakeLists.txt:cmake_minimum_required(VERSION 3.14.2)
src/coreclr/tests/CMakeLists.txt:cmake_minimum_required(VERSION 3.14.2)
src/installer/corehost/CMakeLists.txt:cmake_minimum_required(VERSION 3.14.2)
src/installer/corehost/cli/test/mockhostfxr/CMakeLists.txt:cmake_minimum_required (VERSION 2.6)
src/libraries/Native/Unix/CMakeLists.txt:cmake_minimum_required(VERSION 3.14.2)
src/libraries/Native/Windows/CMakeLists.txt:cmake_minimum_required(VERSION 3.14.2)
src/libraries/Native/Windows/clrcompression/zlib/CMakeLists.txt:cmake_minimum_required(VERSION 2.4.4)
src/mono/CMakeLists.txt:cmake_minimum_required (VERSION 3.10)
src/mono/mono/btls/CMakeLists.txt:cmake_minimum_required (VERSION 2.8.10)
src/mono/netcore/sample/iOS/CMakeLists.txt:cmake_minimum_required(VERSION 3.14.5)
```

Reply:
I agree we should bump the version, but let's do it in another PR :)